tests/nodes.test.js

```diff
@@ -0,0 +1,57 @@
+import { test } from 'zinnia:test'
+import { assertEquals } from 'zinnia:assert'
+import { getNodes } from '../lib/nodes.js'
+
+test('should arweave node by default', async () => {
```

Comment:
What should happen by default? Did you perhaps meant the following?

```suggestion
test('should return arweave.net node by default', async () => {
```